internal/configs/transportserver.go

```diff
@@ -29,10 +49,10 @@ func (tsEx *TransportServerEx) String() string {
 }
 
 // generateTransportServerConfig generates a full configuration for a TransportServer.
-func generateTransportServerConfig(transportServerEx *TransportServerEx, listenerPort int, isPlus bool) version2.TransportServerConfig {
+func generateTransportServerConfig(transportServerEx *TransportServerEx, listenerPort int, tsc *transportServerConfigurator) version2.TransportServerConfig {
```

Comment:
generateTransportServerConfig needs to report warnings, similarly to https://github.com/nginxinc/kubernetes-ingress/blob/master/internal/configs/virtualserver.go#L264

Reply:
We confined Warnings to `generateTransportServerConfig()` as the supporting infrastructure isn't available. Warnings are handled at a higher level and the TransportServer status is updated. Waiting on https://github.com/nginxinc/kubernetes-ingress/pull/1415 for this.